vagrant/Vagrantfile

```diff
@@ -0,0 +1,26 @@
+# -*- mode: ruby -*-
```

Comment:
Sure, it used to be there but I foresaw you saying it should be relegated to its own directory for cleanliness. I can move it back.